app/views/content/train-to-be-a-teacher/promos/_adviser-promo-ske.html.erb

```diff
@@ -4,7 +4,7 @@
   <% end %>
 
   <% promo.right_section(heading: "Get free one-to-one support") do %>
-      <p>Your adviser can help you understand how to do a subject knowledge enhancement course and support you with your teacher training application.</p>
+      <p>Your adviser can help you understand how to do a subject knowledge enhancement course and support you with your teacher training application. </p>
```

Comment:
did you mean to change this @EllieNodder ?

Reply:
GitHub was throwing a tantrum and not working so just added this change in to kick it back to life - will take it back out